packages/x-components/src/x-modules/scroll/components/main-scroll-item.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <component :is="tag" :data-scroll="item.id">
+    <slot />
+  </component>
+</template>
+<script lang="ts">
+  import { Identifiable } from '@empathyco/x-types';
+  import Vue from 'vue';
+  import { Component, Prop } from 'vue-property-decorator';
+  import { State, xComponentMixin } from '../../../components';
+  import { XInject } from '../../../components/decorators/injection.decorators';
+  import { NoElement } from '../../../components/no-element';
+  import UrlHandler from '../../url/components/url-handler.vue';
+  import { scrollXModule } from '../x-module';
+  import MainScroll from './main-scroll.vue';
+  import { FirstVisibleItemObserverKey } from './scroll.const';
+  import { ScrollVisibilityObserver } from './scroll.types';
+
+  /**
+   * Wrapper for elements contained in the {@link MainScroll} that should store/restore its
+   * position.
+   *
+   * @public
+   */
+  @Component({
+    mixins: [xComponentMixin(scrollXModule)],
+    components: { UrlHandler, MainScroll, NoElement }
```

Comment:
Because if you are not careful when writing the doc, webstorm auto-imports them in the component T.T. If I recall correctly Vue's eslint rules detects unused components in its last version.